### PR TITLE
Add missing copyright notices and LICENSE file

### LINKED ISSUE #103: Documentation:- Can you please add LICENSE file ?

### PR DESCRIPTION
**Pull request by ivan4th**
In particular, add copyright notices to E2E tests
Fixes #103 (missing LICENSE file)

### CODE REVIEW

File: test/e2e/framework/expect.go [hunk 1]
```diff
@@ -1,3 +1,19 @@
+// expect.go - 3GPP TS 29.244 GTP-U UP plug-in
+//
+// Copyright (c) 2020-2021 Travelping GmbH
```
Comment (RoadRunnr) [inline test/e2e/framework/expect.go:3]: you don't need to put in a date range, using simply:

```
// Copyright (c) 2020 Travelping GmbH
```

File: upf/upf_pfcp_api.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright(c) 2018 Travelping GmbH.
+ * Copyright(c) 2018-2021 Travelping GmbH.
```
Comment (RoadRunnr) [inline upf/upf_pfcp_api.h:2]: no need for year ranges

File: upf/upf_ipfilter.c [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright(c) 2018 Travelping GmbH.
+ * Copyright(c) 2021 Travelping GmbH.
```
Comment (RoadRunnr) [inline upf/upf_ipfilter.c:2]: updating the copyright is also no needed and only bloats the diff

**Review by RoadRunnr — CHANGES_REQUESTED** [review]
Updating copyright dates is not needed. Existing copyright message should be left alone.

File: upf/upf_ipfilter.c (hunk 3 above)
Comment (ivan4th) [reply upf/upf_ipfilter.c]: Removed the updates in `upf/` from the patch

**Review by RoadRunnr — APPROVED** [review]
LGTM

**Review by sergeymatov — APPROVED** [review]

**Pull request merged**